### ISSUE TITLE
bug in create policy select_invited

### ISSUE THREAD

**Issue by jmarca** [graphile/starter#148]
I believe the select_invited policy is incorrect.

https://github.com/graphile/starter/blob/1b912b768911e2b0c94eef4eebcdfed6d036f8bd/%40app/db/migrations/committed/000001.sql#L1260

It should be 

```sql
create policy select_invited on app_public.organization_invitations
```

Not sure how the test here is passing:

https://github.com/graphile/starter/blob/ab3a443c63ede32b155c0d8acb75f2171fbe9fdc/%40app/db/__tests__/app_public/functions/invite_to_organization.test.ts#L118

but my pg_prove test fails.  I've copied the setup and the test, inserting plain sql selects so that you should be able to just copy paste into a test db to check.

```sql
begin;
-- create some test data
-- user
INSERT INTO app_public.users (username,name) VALUES ('jmarca', 'James E. Marca');

-- email
WITH uid(id) as (select id from app_public.users where username='jmarca')
insert into app_public.user_emails (user_id, email, is_verified)
   select uid.id, 'james@activimeowtricks.com', true from uid;

-- organization
INSERT INTO app_public.organizations (slug,name) VALUES ('Marca','The Marca Family');

-- invitation to membership
WITH uid(id) as (select id from app_public.users where username='jmarca'),
     oid(id) as (select id from app_public.organizations where slug='marca'::citext)
insert into app_public.organization_invitations (user_id, organization_id)
   select uid.id as user_id, oid.id as organization_id
   from uid
   join oid on (true);

-- ...
with uid(id) as (select id from app_public.users where username='jmarca')
insert into app_private.sessions (user_id)
   select uid.id  from uid;
-- fake jwt claims
with
uid(id) as (select id from app_public.users where username='jmarca'),
sid(uuid) as (select uuid from app_private.sessions s join uid u on (u.id=s.user_id))
select set_config('jwt.claims.session_id', sid.uuid::text, true)
from sid;

select organization_id from app_public.organization_invitations;
--           organization_id            
-- ----------------------------------------
--  c79f5e65-f924-4483-88a1-3136ab3061b2
-- (1 row)

SET ROLE :DATABASE_VISITOR;

-- failing test
-- select isnt_empty('select organization_id from app_public.organization_invitations',
--                   'Should select from invitations if is session');
-- ... 

select organization_id from app_public.organization_invitations;
--  organization_id 
-- -----------------
-- (0 rows)

rollback;
```

If I change the policy to apply to organization_invitations, my test passes (because the visitor role can now select its own invitations from organization_invitations, of course).

**Comment by benjie** [comment]
This policy is correct - you're allowed to see all memberships to an organization if you're invited to the organization (i.e. you're allowed to select organization_memberships if you're invited -> select_invited). We haven't added a view for seeing your invitations yet, so I've not worried about exposing that functionality - the email flow should work in general.

If you'd like to add an "invitations" view (e.g. under /settings/), that would be a welcome PR :+1:; I deemed it non-essential when I wrote the rest so didn't implement it.

**Comment by jmarca** [comment]
Hmm, it seems I was misunderstanding.

So I fiddled with my sqitch version a bit more and I think I get what you're saying.

The purpose of this policy is to allow a newly invited user to see all of the current members of a group, correct?  Use case being you want to see what you're joining.  If a dog is invited to join a group of household pets and the only current members are cats, the dog probably will not choose to join.

All works for me.  Sorry for the noise.

(And staring harder at the test I referenced above, I see the line `await becomeRoot(client);` just above the query that will fail if not "superuser" in the db.)
